specification/web/resource-manager/Microsoft.Web/2016-09-01/AppServiceEnvironments.json

```diff
@@ -1,3006 +1,3222 @@
 {
-  "swagger": "2.0",
-  "info": {
-    "version": "2016-09-01",
-    "title": "AppServiceEnvironments API Client"
-  },
-  "host": "management.azure.com",
-  "schemes": [
-    "https"
-  ],
-  "paths": {
-    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/hostingEnvironments": {
-      "get": {
-        "tags": [
-          "AppServiceEnvironments"
-        ],
-        "summary": "Get all App Service Environments for a subscription.",
-        "description": "Get all App Service Environments for a subscription.",
-        "operationId": "AppServiceEnvironments_List",
-        "produces": [
-          "application/json",
-          "text/json"
-        ],
-        "parameters": [
-          {
-            "$ref": "#/parameters/subscriptionIdParameter"
-          },
-          {
-            "$ref": "#/parameters/apiVersionParameter"
-          }
-        ],
-        "responses": {
-          "200": {
-            "description": "OK",
-            "schema": {
-              "$ref": "#/definitions/AppServiceEnvironmentCollection"
-            }
-          }
-        },
-        "x-ms-pageable": {
-          "nextLinkName": "nextLink"
-        }
+ "swagger": "2.0",
```

Comment:
Changing formatting of the document is unnecessary changes and it's really hard to review. Could you format the document as it was before?